tests/integration/destroy/test_destroy_command_integ.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+import time
+import uuid
+from subprocess import Popen, PIPE
+from unittest import skipIf
+
+import boto3
+from botocore.exceptions import ClientError
+
+from samcli.commands.destroy import verify_stack_exists
+from tests.integration.destroy.test_destroy_integ_base import DestroyIntegBase
+
+LOG = logging.getLogger(__name__)
+
+# Publish tests require credentials and Travis will only add credentials to the env if the PR is from the same repo.
```

Comment:
destroy tests, also our build system is all appveyor now, so this may need to change.

Reply:
I updated the tests to use the AppVeyor format